lib/improver/cli/nowcast_optical_flow.py

```diff
@@ -88,57 +86,25 @@ def main(argv=None):
                         help="Number of iterations to perform in enforcing "
                         "smoothness constraint for optical flow velocities.")
 
-    # AdvectField options
-    parser.add_argument("--extrapolate", action="store_true", default=False,
-                        help="Optional flag to advect current data forward to "
-                        "specified lead times.")
-    parser.add_argument("--max_lead_time", type=int, default=360,
-                        help="Maximum lead time required (mins).  Ignored "
-                        "unless '--extrapolate' is set.")
-    parser.add_argument("--lead_time_interval", type=int, default=15,
-                        help="Interval between required lead times (mins). "
-                        "Ignored unless '--extrapolate' is set.")
-
     args = parser.parse_args(args=argv)
 
     # Load Cubes and JSON
     attributes_dict = load_json_or_none(args.json_file)
     original_cube_list = load_cubelist(args.input_filepaths)
-    oe_cube = load_cube(args.orographic_enhancement_filepaths,
-                        allow_none=True)
+    oe_cube = load_cube(args.orographic_enhancement_filepaths, allow_none=True)
 
     # Process
-    forecast_cubes, u_and_v_mean = process(
+    result = process(
         original_cube_list, oe_cube, attributes_dict, args.ofc_box_size,
-        args.smart_smoothing_iterations, args.extrapolate,
-        args.max_lead_time, args.lead_time_interval)
+        args.smart_smoothing_iterations)
 
     # Save Cubes
-    for wind_cube in u_and_v_mean:
-        file_name = generate_file_name(wind_cube)
-        save_netcdf(wind_cube, os.path.join(args.output_dir, file_name))
-
-    # advect latest input data to the required lead times
-    if args.extrapolate:
-        if args.nowcast_filepaths:
-            if len(args.nowcast_filepaths) != len(forecast_cubes):
-                raise ValueError("Require exactly one output file name for "
-                                 "each forecast lead time")
-
-        for i, cube in enumerate(forecast_cubes):
-            # save to a suitably-named output file
-            if args.nowcast_filepaths:
-                file_name = args.nowcast_filepaths[i]
-            else:
-                file_name = os.path.join(
-                    args.output_dir, generate_file_name(cube))
-            save_netcdf(cube, file_name)
+    save_netcdf(result, args.output_filepath)
 
 
 def process(original_cube_list, orographic_enhancement_cube=None,
             attributes_dict=None, ofc_box_size=30,
-            smart_smoothing_iterations=100, extrapolate=False,
-            max_lead_time=360, lead_time_interval=15):
+            smart_smoothing_iterations=100):
```

Comment:
The docstring for this method needs to be updated to remove the references to extrapolation, as this function is only computing the optical flow now.

Reply:
Done.